### PR TITLE
issue #6 address search

### PR DESCRIPTION
**Pull request by holarora**
## Backlog チケット URL

​
### 関連 Backlog チケット URL (あれば)

​
## 動作確認

​
## レビュー観点

​
## 悩んでいること (あれば)

​

### CODE REVIEW

**Comment by holarora** [comment]
https://github.com/pj100-team/react-frontend-tutorial-v2/assets/138124278/34126565-f332-4871-9149-25baf1560a00

This is a preview of the address search project of issue #6.

File: src/app/components/common/address_search/addresssearch.css [hunk 1]
```diff
@@ -0,0 +1,19 @@
+.error-message {
+    display: none;
```
Comment (kihara-masaki) [inline src/app/components/common/address_search/addresssearch.css:2]: why did you set this property?

if you don't have reason, I recommend you to delete it.

File: src/app/components/common/address_search/addresssearch.css [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    color: red;
+    border: red solid 3px;
```
Comment (kihara-masaki) [inline src/app/components/common/address_search/addresssearch.css:4]: I think this one is also unnecessary.

**Comment by kihara-masaki** [comment]
I think below behavior is better than before. 
If you like this one, I share detail code. 

https://github.com/pj100-team/react-frontend-tutorial-v2/assets/103478597/024fac5f-49d9-4c2d-aebc-56cc5972b129

File: src/app/components/common/address_search/addresssearch.tsx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+        setIsLoading(true);
+        await axios
+          .get("https://zipcloud.ibsnet.co.jp/api/search", {
+            params: {
+              zipcode: text,
+            },
+          })
+          .then((res) => {
+            setAddress({
+                address1: res.data.results[0].address1,
```
Comment (kihara-masaki) [inline src/app/components/common/address_search/addresssearch.tsx:30]: To specify magic number is not good.
If you can't understand what I said, please inform me.

**Review by kihara-masaki — APPROVED** [review]

**Pull request merged**